src/contracts/Staking.sol

```diff
@@ -116,14 +126,27 @@ contract Staking is Ownable {
     }
     Epoch public epoch;
 
+    struct Claim {
```

Comment:
We _could_ probably tightly pack this struct, maybe something we can save as a TODO / GH issue to look at in the end for gas optimization.